dev/ci/ci-metacoq.sh

```diff
@@ -5,4 +5,8 @@ ci_dir="$(dirname "$0")"
 
 git_download metacoq
 
+export COQEXTRAFLAGS='-native-compiler no'
+# examples/Makefile.coq.local is overriding COQEXTRAFLAGS
+sed -i.bak 's/\(COQEXTRAFLAGS=.*\)/\1 -native-compiler no/' \
+  ${CI_BUILD_DIR}/metacoq/examples/Makefile.coq.local
```

Comment:
CC @mattam82

Reply:
Ok, I'll see what I can do

Reply:
Just replacing `=` with `+=` should be fine.

Reply:
I changed the Makefile to use `+=` instead

Reply:
Thanks!